### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Rahaman2003**
->Create Assignment_4  
->Implement Banking system
->using the random library in OTP, Credit card number
->using if, else condition in login account
->using append function

### CODE REVIEW

File: assignment_1.py [hunk 1]
```diff
@@ -0,0 +1,251 @@
+Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] on win32
```
Comment (praveen-pb) [inline assignment_1.py]: I guess you have copy pasted the code from terminal. Use VS Code to create and write python code. That's the correct way to do.

File: assignment_1.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+vegetables=['onion','carrot','ginger']
+fruits.extend(vegetables)
+print(fruits)
+"""
+['banana', 'graps', 'apple', 'lemon', 'onion', 'carrot', 'ginger']
+"""
+"""
+#index
+fruits.index()
+Traceback (most recent call last):
```
Comment (praveen-pb) [inline assignment_1.py]: Files should not contain these errors. Use VS Code to write code. So these errors will be rectified when you run the code.

File: assignment_1.py [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+print(fruits)
+"""
+['banana', 'guava', 'graps', 'apple', 'lemon', 'onion', 'carrot', 'ginger']
+['banana', 'guava', 'graps', 'apple', 'lemon', 'onion', 'carrot', 'ginger']
+['banana', 'guava', 'graps', 'apple', 'lemon', 'onion', 'carrot', 'ginger']
+"""
+"""
+#pop
+fruits.pop('guava')
+Traceback (most recent call last):
```
Comment (praveen-pb) [inline assignment_1.py]: Same here

File: assignment_1.py [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+#count
+tuple.count('dhoni')
+# 1
+#index
+"""
+tupke.index('sachin')
+Traceback (most recent call last):
+  File "<pyshell#53>", line 1, in <module>
+    tupke.index('sachin')
+NameError: name 'tupke' is not defined. Did you mean: 'tuple'?
```
Comment (praveen-pb) [inline assignment_1.py]: These things should not be in python file. this will create bad remarks.

File: assignment_1.py
Comment (praveen-pb) [inline assignment_1.py:1]: Please update this entire file. This got lot of errors. remove those and update as python file. DO NOT copy paste from terminal

File: assignment_3.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+"""     
+rahaman
+19
+"""
+
+
+
+#creating class in employee details
+class Employee:
```
Comment (praveen-pb) [inline assignment_3.py]: Add more logics to the code. use type, isintance, list, tuple, dict. create like a story. currently you are just printing the data. this won't be enough to show to companies.

File: assignment_3.py [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+"""
+19 =emp_age
+zoho =emp_company
+50000 =emp_salary
+fullstack developer =emp_work
+variable x is not defined 
+"""
+
+#create the try and expect
+try: #try block for code the error
```
Comment (praveen-pb) [inline assignment_3.py]: Try exception block should be inside the function. I have mentioned this during the class. follow as per that.

File: assignment_3.py [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+           print("friday")
+         case 6:
+           print("saterday")
+         case 7:
+           print("invalid date")
+           
+          
+#anothe example of match case
+grade =1
+match grade:
```
Comment (praveen-pb) [inline assignment_3.py]: match case should be inside a function. create a class and add it inside any of the function

File: assignment_4.py [hunk 8]
```diff
@@ -0,0 +1,77 @@
+from random import randint
+
+class BankingSystem:
+    def __init__(self):
+        self.creaditcard_number=[]
```
Comment (praveen-pb) [inline assignment_4.py]: don't make spelling mistakes in code.

File: assignment_4.py
Comment (praveen-pb) [inline assignment_4.py:1]: You have initiated well, but try to add more logics. this is not enough. try creating account with some input from user. have some amount as bank balance, give loan, receive interest. handle multiple customers. this way you can have large codebase.

**Review by praveen-pb — COMMENTED** [review]
Code base needs to be increased. please check out our recorded videos where I have talked about the common mistakes in assignments.

File: assignment_1.py (hunk 1 above)
Comment (Rahaman2003) [reply assignment_1.py]: sorry sir, first time upload this code github for my mistake ,I rectify for my mistake

**Pull request merged**